CHANGELOG.md

```diff
@@ -43,6 +45,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
   - Move `proofs` package to `com.exonum.binding.common` package. (#469)
   - Move `serialization` package to `com.exonum.binding.common` package. (#469)
 - Replace tree proof with flat proof in `ProofMapIndexProxy`. (#478)
+- `ListProofValidator` separated to `ListProofStructureValidator` 
```

Comment:
I'd better replace it with a user-oriented entry:

`ProofListIndexProxy#getProof` and `ProofListIndexProxy#getRangeProof` to return
`UncheckedListProof` instead of `ListProof`. The latter is renamed into `ListProofNode`
and may be accessed through `UncheckedListProof#getRootProofNode`